src/webapp/components/member-selector/MemberSelector.tsx

```diff
@@ -0,0 +1,166 @@
+
+export type MemberOption = {
+    value: string;
+    label: string;
+    disabled?: boolean;
+    workPosition: string;
+    phone: string;
+    email: string;
+    status: string;
+    src?: string;
+    alt?: string;
+};
+
+type MemberSelectorProps = {
+    id: string;
+    selected?: string;
+    onChange: (value: string) => void;
+    options: MemberOption[];
+    label?: string;
+    placeholder?: string;
+    disabled?: boolean;
+    helperText?: string;
+    errorText?: string;
+    error?: boolean;
+    required?: boolean;
+};
+
+export const MemberSelector: React.FC<MemberSelectorProps> = React.memo(props => {
+    const {
+        id,
+        label,
+        placeholder = "",
+        selected = "",
+        onChange,
+        options,
+        disabled = false,
+        helperText = "",
+        errorText = "",
+        error = false,
+        required = false,
+    } = props;
+
+    const selectedMember = useMemo(() => {
+        return options.find(option => option.value === selected);
+    }, [options, selected]);
+
+    return (
+        <ComponentContainer>
+            <SelectorContainer>
+                <Selector
+                    id={id}
+                    placeholder={placeholder}
+                    label={label}
+                    selected={selected}
+                    onChange={value => onChange(value)}
+                    options={options}
+                    helperText={helperText}
+                    errorText={errorText}
+                    error={error}
+                    required={required}
+                    disabled={disabled}
+                />
+            </SelectorContainer>
+            {selectedMember && (
+                <AvatarContainer>
+                    <AvatarCard
+                        avatarSize="medium"
+                        alt={selectedMember?.alt}
+                        src={selectedMember?.src}
+                    >
+                        <Container>
+                            <Content>
+                                <TextBold>{selectedMember?.label}</TextBold>
+                                <Text>{selectedMember?.workPosition}</Text>
+                            </Content>
+                            <Content>
+                                <Text>{selectedMember?.phone}</Text>
+                                <StyledLink href={`mailto:${selectedMember?.email}`}>
+                                    {selectedMember?.email}
+                                </StyledLink>
+                            </Content>
+                            <div>
+                                <TextBold>{i18n.t("Status: ", { nsSeparator: false })}</TextBold>
+                                <Text>{selectedMember?.status}</Text>
+                            </div>
+                        </Container>
+                    </AvatarCard>
+                </AvatarContainer>
+            )}
+        </ComponentContainer>
```

Comment:
[subjective] some blank lines on elements at the same level to let the component breath